apps/web/src/app/components/Alert.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+import { motion, AnimatePresence } from "framer-motion";
+import { classed } from "~/utils/classed";
+import { create } from "zustand";
+import { shallow } from "zustand/shallow";
+import { nanoid } from "nanoid";
+
+const AlertContainer = classed.ul(
+  "fixed top-3 inset-x-0 flex flex-col justify-center items-center z-50",
+);
+
+const AlertTemplate = classed(
+  motion.li,
+  "text-white",
+  "flex items-center cursor-pointer",
+  "p-3 rounded-wavy max-w-[500px]",
```

Comment:
this component looks doesn't look like its migrated? we shouldn't be using `rounded-wavy`